data/rule_data.yml

```diff
@@ -9,6 +9,7 @@ rule_data:
 
   allowed_step_image_registry_prefixes:
   - quay.io/redhat-appstudio/
+  - quay.io/redhat-appstudio-tekton-catalog/
```

Comment:
I don't think this is doing what you think it's doing. 

This list is used when validating new Task definitions to make sure they use images that come from a certain location. It is not used when validating images built in Konflux.

Sync up with Lance. We were chatting about an issue similar to what you mentioned.